openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,13 +570,17 @@ OpenAssessment.ResponseView.prototype = {
     },
 
     /**
-     Set the image URL, or retrieve it.
+     Set the file URL, or retrieve it.
      **/
-    imageUrl: function() {
+    fileUrl: function() {
         var view = this;
-        var image = $('#submission__answer__image', view.element);
+        var file = $('#submission__answer__file', view.element);
         view.server.getDownloadUrl().done(function(url) {
-            image.attr('src', url);
+            if (file.prop("tagName") == "IMG") {
```

Comment:
It's a good idea to always use triple equals; please see
http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons

On Tue, Oct 13, 2015 at 2:47 PM, Pan Luo <notifications@github.com> wrote:

> In openassessment/xblock/static/js/src/lms/oa_response.js
> <https://github.com/edx/edx-ora2/pull/708#discussion_r41907806>:
>
> >          view.server.getDownloadUrl().done(function(url) {
> > -            image.attr('src', url);
> > +            if (file.prop("tagName") == "IMG") {
>
> I don't think there is any issue when comparing string. Though I'll change
> it to be more clear.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-ora2/pull/708/files#r41907806>.
>